gen/generator.go

```diff
@@ -231,7 +233,7 @@ func GetTestValue(name string, model Model) string {
 					childValue := GetOverwriteAttributeValue(child.PkgName, childKey, fmt.Sprintf("%s_1", childKey), "default", 0, model.Definitions).(string)
 					properties = append(properties, getChildTarget(model, childKey, childValue, false))
 				} else if property.PropertyName == "tDn" {
-					return getChildTarget(model, child.PkgName, "target_dn_0", true)
+					return getChildTarget(model, child.PkgName, "", true)
```

Comment:
Was this change intended? I don't see that it has changed any other test files so it looks good to me.
I noticed the tDn of infraRsAccBndlSubgrp is set to an empty string hence double checking to make sure this change was also intended.